Ghidra/Processors/ARM/data/languages/ARMTHUMBinstructions.sinc

```diff
@@ -2877,24 +2877,24 @@ control: "control" 			is epsilon {}
 
 @if defined(VERSION_7M)
 
-msripsr: "i"	is thc0000=1 { }
-msripsr: 		is thc0000=0 { }
-msrepsr: "e"	is thc0101=1 { }
-msrepsr: 		is thc0101=0 { }
-msrapsr: 		is thc0202=1 { }
-msrapsr: "a"	is thc0202=0 & Rn0003 {
+msripsr: "i"	is Rn0003; thc0000=1 { }
```

Comment:
Question to Ghidra developers: Except for line 2885 we don't actually need Rn0003 here; but the compiler forces me to put something before the semicolon. Is there a way to put something empty here?

Reply:
It just needs to know that it has to consume a token of a certain size. `op0` might be more clear, since it basically hints "consume the entire token and match anything", however since one does need Rn0003, I think the way you have addressed it is fine.